src/panels/NightSkyPanel/NightSkyPanel.tsx

```diff
@@ -0,0 +1,50 @@
+import { Container, Divider, Group, Space, Tabs } from '@mantine/core';
+import { useElementSize } from '@mantine/hooks';
+
+import { NightSkyLocationTab } from './tabs/NightSkyLocationTab';
+import { NightSkyMarkingsTab } from './tabs/NightSkyMarkingsTab';
+import { NightSkySolarSystemTab } from './tabs/NightSkySolarSystemTab';
+import { NightSkyStarsTab } from './tabs/NightSkyStarsTab';
+import { NightSkySunTab } from './tabs/NightSkySunTab';
+import { NightSkyTimeTab } from './tabs/NightSkyTimeTab';
+
+export function NightSkyPanel() {
+  const { ref } = useElementSize();
+
+  return (
+    <Container fluid>
```

Comment:
Change this to a `Box` component to get rid of the extra padding and make styling more consistent with other panels: 
```suggestion
    <Box>
```